File: UBL/CommonAggregateComponents/ResultOfVerificationType.cs
using Hilma.UBL.Attributes;
using Hilma.UBL.UnqualifiedDataTypes;

namespace Hilma.UBL.CommonAggregateComponents
{
  [Contract]
  public class ResultOfVerificationType
  {
    public IdentifierType ValidatorID { get; set; }
    public CodeType ValidationResultCode { get; set; }
    public DateType ValidationDate { get; set; }
    public TimeType ValidationTime { get; set; }
    public string ValidateProcess { get; set; }
    public string ValidateTool { get; set; }
    public string ValidateToolVersion { get; set; }
    public PartyType SignatoryParty { get; set; }

  }

}

File: Espd/typings/ResultOfVerificationType.ts



import { IdentifierType } from './IdentifierType'
import { CodeType } from './CodeType'
import { DateType } from './DateType'
import { TimeType } from './TimeType'
import { PartyType } from './PartyType'

import assign from 'lodash.assign'

export class ResultOfVerificationType { 
    public constructor(init?:Partial<ResultOfVerificationType>) {
        assign(this, init)
    } 
    
    
    validatorID?: IdentifierType
    validationResultCode?: CodeType
    validationDate?: DateType
    validationTime?: TimeType
    validateProcess?: string
    validateTool?: string
    validateToolVersion?: string
    signatoryParty?: PartyType
}